src/components/App/AppHeader/AppHeader.jsx

```diff
@@ -336,7 +336,7 @@ export default function AppHeader() {
           <Button onClick={openFile}>Open</Button>
           <Button onClick={saveFile}>Save</Button>
           <Button onClick={saveFileAs}>Save As</Button>
-          {context.isThingModel && <Button onClick={openConvertTmDialog}>Convert To TD</Button>}
+          {(context.showConvertBtn || context.isThingModel) && <Button onClick={openConvertTmDialog}>Convert To TD</Button>}
```

Comment:
Could we maybe remove one of `showConvertBtn` and ``isThingModel``. 
I think one flag would be enough to determine whether it is a TM?

Reply:
The thing is that `context.isThingModel` is initiated (and updated) only on editor's `onChange` event. So, when a TM is created with the New button, that process is not triggered until something is changed in the editor (e.g. typing happened) thus "Convert to TD" button is not shown initially. I agree that semantically one value for this would be enough, however, it requires refactoring of the editor's `onChange` event handler, where pretty much logic is happening - https://github.com/eclipse/editdor/blob/d5dcf1bf287015d4cf2e39e83c1e15a1d8e1ba37/src/components/Editor/Editor.jsx#L115 and which requires a more granular understanding of what's happening there. That's why I simply didn't touch that and introduced this new variable.

Reply:
I can live with this for now. 
But maybe for the next change we do here we can add something that triggers the `onChange` event when a TM is created.

Reply:
Agree. I thought about it as well, but it turned out to be not so trivial.